src/main/res/values/strings.xml

```diff
@@ -45,23 +45,23 @@
   <string name="downloading_format">Downloading %s</string>
   <string name="edit_repository">Edit repository</string>
   <string name="empty_search_summary">No matching applications found</string>
-  <string name="error_downloading_format">Error downloading %s</string>
-  <string name="error_syncing_format">Error syncing %s</string>
-  <string name="error_validating_format">Error validating %s</string>
+  <string name="error_downloading_format">Could not download %s</string>
+  <string name="error_syncing_format">Could not sync %s</string>
+  <string name="error_validating_format">Could not validate %s</string>
   <string name="fingerprint">Fingerprint</string>
-  <string name="finished_downloading_format">Finished downloading %s</string>
+  <string name="finished_downloading_format">Downloaded %s</string>
   <string name="http_error_description">Invalid server response.</string>
   <string name="http_proxy">HTTP proxy</string>
   <string name="ignore_all_updates">Ignore all updates</string>
   <string name="ignore_this_update">Ignore this update</string>
   <string name="incompatible_features_description">Missing features:</string>
-  <string name="incompatible_older_description">This version is older than installed on your device.
-    You\'ll have to uninstall the application to proceed.</string>
+  <string name="incompatible_older_description">Older than the version installed on your device.
+    Uninstall the current application to install the older version.</string>
```

Comment:
(old conversation removed from this change)

> The user only needs to know "Uninstall the current application before installing an older version"

You're not right. User needs to know why this version is incompatible. It's because it's older than installed one. And then solution is suggested.

> Problem 1, it isn't really incompatible

This is how it's designed right now, either the application compatible (so it can be installed), or not. It's actually "incompatible" in sense it's "incompatible to the current package management system state", package manager won't install it.

> Problem 2, "Older than installed" is an error

I don't mind changing to "Older than installed one". You suggested to replace the whole phrase to "Older than installed one" before, while I think "This version is older than installed one" should be used.

>  Problem 3, it is too long.

There are even longer messages :) It's OK to place long messages into alert dialogs.